v2g-liberty-cards/src/charger-settings-card.ts

```diff
@@ -93,6 +94,35 @@ export class ChargerSettingsCard extends LitElement {
     `;
   }
 
+  private _isV2GlibertyLoadbalancerEnabled(): boolean {
+    const ql = this._quasarLoadbalancerLimit?.state.toLowerCase();
```

Comment:
Is it necessary to use `?` here. I prefer to avoid using this, as the variable should be initialized properly.

Reply:
The lowercase was a result of a (wrong) init from the code where it was set to "Unknow", i'll remove it.